RecoTauTag/Configuration/test/QCD-validation.txt

```diff
@@ -0,0 +1,5 @@
+/store/relval/CMSSW_8_1_0_pre11/RelValQCD_Pt_80_120_13/GEN-SIM-RECO/81X_mcRun2_asymptotic_Candidate_2016_08_30_11_31_55-v1/00000/24F576D4-4D74-E611-8898-0025905A48BA.root
```

Comment:
These files are only used for our internal tests. They are anyhow obsolete, once the validation samples are deleted form relval at CERN. But I've added some more intuitive dummy files for the next commit.